addons/account/views/account_journal_dashboard_view.xml

```diff
@@ -334,12 +334,12 @@
                             <div class="row">
                                 <div class="col overflow-hidden text-left">
                                     <a type="object" t-if="journal_type == 'sale'" name="open_action"
-                                    context="{'search_default_unpaid':1}" id="account_dashboard_sale_pay_link">
+                                    context="{'search_default_unpaid':1, 'search_default_posted':1}" id="account_dashboard_sale_pay_link">
                                         <t t-esc="dashboard.number_waiting"/> Unpaid Invoices
                                     </a>
 
                                     <a type="object" t-if="journal_type == 'purchase'" name="open_action"
-                                    context="{'search_default_unpaid':1}" id="account_dashboard_purchase_pay_link">
+                                    context="{'search_default_unpaid':1, 'search_default_posted':1}" id="account_dashboard_purchase_pay_link">
```

Comment:
```suggestion
                                    context="{'search_default_unpaid':1, 'search_default_partial':1, 'search_default_posted':1}" id="account_dashboard_purchase_pay_link">
```

Reply:
I think you mean 'search_default_in_payment':1